include/odp/api/spec/classification.h

```diff
@@ -206,6 +213,43 @@ typedef struct odp_cls_cos_param {
 
 	/** Drop policy associated with CoS */
 	odp_cls_drop_t drop_policy;
+
+	/* Random Early Detection (RED)
+	 * Random Early Detection is enabled to initiate a drop probability
```

Comment:
Missing Doxygen tag /**

Run 'make doxygen-doc' locally before pushing and correct all doxygen warnings.

Reply:
Okay. 
@muvarov Travis does not check doxygen?

Reply:
@bala-manoharan there was error in check script, I fixed that.